backend/battle/urls_api.py

```diff
@@ -6,4 +6,5 @@
     path("battle/<int:pk>/", endpoints.BattleDetail.as_view(), name="battle-detail"),
     path("battle/create/", endpoints.BattleCreate.as_view(), name="battle-create"),
     path("team/create/", endpoints.TeamCreate.as_view(), name="team-create"),
+    path("user/", endpoints.LoggedUser.as_view(), name="logged-user"),
```

Comment:
Leaving this one comment since I already reviewed this in another PR: I think this should be prefixed with `api/` and the name of View could be improved, I suggest: `CurrentUserAPI`

Reply:
Thanks, Rebeca :) In the `pokebattle/urls.py` I set the prefix to all the URLs related to `urls_api`

Reply:
like here 
![image](https://user-images.githubusercontent.com/53096507/130372220-bb7780eb-deb3-41b2-9da9-c0174c72e299.png)